### PR TITLE
Fix CallbackServerTest

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,3 +50,4 @@ src/java/us/kbase/mobu/git.properties
 /a_SimpleModule_for_unit_testing_java/
 /a_SimpleModule_for_unit_testing_python/
 /a_SimpleModule_for_unit_testing_perl/
+/ASimpleModule_for_unit_testingPythonError/
```
Comment (briehl) [inline .gitignore:53]: I'm not seeing where this gets created or referenced.

File: src/java/us/kbase/mobu/tester/test/CallbackServerTest.java [hunk 2]
```diff
@@ -405,7 +405,8 @@ public void maxJobs() throws Exception {
             res.callMethod("njs_sdk_test_1.run", params, "dev");
         } catch (ServerException se) {
             assertThat("incorrect error message", se.getLocalizedMessage(),
-                    is("No more than 10 concurrently running methods are allowed"));
+                    // you've got to be shitting me right now with this quoting shit wtf
+                    is("\"'No more than 10 concurrently running methods are allowed'\""));
```
Comment (briehl) [inline src/java/us/kbase/mobu/tester/test/CallbackServerTest.java:409]: So the CBS makes the string
"'No more than 10....'"?
With single and double quotes inline?
That's something.

File: src/java/us/kbase/mobu/tester/test/CallbackServerTest.java [hunk 3]
```diff
@@ -510,8 +511,12 @@ public void badRelease() throws Exception {
...
                  "release: 'No module version found that matches your criteria!'");
 
-        //this is the newest git commit and was registered in dev but 
-        //then the previous git commit was registered in dev
+        // this git commit was registered to dev but 
+        // then the previous git commit was registered to dev
+        // Note 7 years later - I don't understand what the above comment means.
+        // Dev versions aren't deleted if you register a new dev version AFAICT
+        // That being said, the test passes.
+        // Maybe the catalog used to replace dev versions?
```
Comment (briehl) [inline src/java/us/kbase/mobu/tester/test/CallbackServerTest.java:519]: Yeah, I remember that it used to. There was only a single dev version at a time, and the previous was wiped. Is it not that way anymore?

Comment (MrCreosote) [reply src/java/us/kbase/mobu/tester/test/CallbackServerTest.java:519]: It doesn't seem to be, the test passes with dev versions that were replaced with different dev versions

File: .gitignore (hunk 1 above)
Comment (MrCreosote) [reply .gitignore:53]: https://github.com/kbase/kb_sdk_plus/blob/8383d43b829146b7efc09eade559d9008e88b6f5/src/java/us/kbase/mobu/tester/test/ModuleTesterTest.java#L119

File: src/java/us/kbase/mobu/tester/test/CallbackServerTest.java (hunk 3 above)
Comment (briehl) [reply src/java/us/kbase/mobu/tester/test/CallbackServerTest.java:519]: 🤷

**Review by briehl — APPROVED** [review]
LGTM

**Pull request merged**